### PR TITLE
doc:  fix metric guide

### LINKED ISSUE #331: multi cluster monitoring guide does not work
following this guide https://kurator.dev/docs/fleet-manager/metric-plugin/

The attached cluster does not become ready. And all the following step can not work

### PR DESCRIPTION
**Pull request by zirain**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #331 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: docs/content/en/docs/fleet-manager/metric-plugin.md [hunk 1]
```diff
@@ -22,6 +22,12 @@ Fleet's multi cluster monitoring is built on top [Prometheus](https://prometheus
 
 1. Kurator metric depends on [Thanos](https://thanos.io), [Object Storage](https://thanos.io/tip/thanos/storage.md/) is required for Thanos. In the task, [Minio](https://min.io/) is used, setup by the [installation guide](/docs/setup/install-minio).
 
+1. Running following command to attach cluster secrets.
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/metric-plugin.md:25]:
```suggestion
1. Running the following command to create two secrets to access attached clusters.
```

File: docs/content/en/docs/fleet-manager/metric-plugin.md [hunk 2]
```diff
@@ -22,6 +22,12 @@ Fleet's multi cluster monitoring is built on top [Prometheus](https://prometheus
...
+
+```console
+kubectl create secret generic kurator-member1 --from-file=kurator-member1.config=/root/.kube/kurator-member1.config
+kubectl create secret generic kurator-member2 --from-file=kurator-member2.config=/root/.kube/kurator-member2.config
+```
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/metric-plugin.md:30]: After L38 we can also explain that what we have installed by creating the fleet

File: docs/content/en/docs/fleet-manager/policy.md [hunk 3]
```diff
@@ -34,6 +41,16 @@ After a while, we can see the fleet is `ready`:
 kubectl wait fleet quickstart --for='jsonpath='{.status.phase}'=Ready'
 ```
 
+Thanos and Grafana are installed correctly:
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/policy.md:44]: ? Shouldnot this be in the above doc, here it is installing policy engine

Comment (zirain) [reply docs/content/en/docs/fleet-manager/policy.md]: my mistake, fixed.

**Review by hzxuzhonghu — APPROVED** [review]

**Pull request merged**